dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/SubSnpCoreFieldsToVariantProcessorTest.java

```diff
@@ -134,14 +138,18 @@ public void testMultiallelicReverse() throws Exception {
                                      1766472L, 1766472L, Orientation.FORWARD, LocusType.LONGER_ON_CONTIG, "4", 91223961L,
                                      91223961L, "GTA", "GTA", "T", "TAC/A/CC", "NC_006091.4:g.91223961T>A", 91223961L,
                                      91223961L, Orientation.FORWARD, "NT_455866.1:g.1766472T>A", 1766472L,
-                                     1766472L, Orientation.FORWARD, DBSNP_BATCH);
+                                     1766472L, Orientation.FORWARD, "TAC|A, A|CC", DBSNP_BATCH);
         Variant variant = new Variant("4", 91223961L, 91223963L, "GTA", "T");
         variant.setMainId("rs" + 13677177L);
         variant.setDbsnpIds(TestUtils.buildIds(26201546L, 13677177L));
         Map<String, String> attributes = Collections.singletonMap(DBSNP_BUILD_KEY, String.valueOf(DBSNP_BUILD));
+
+        List<Map<String, String>> genotypeMap = new ArrayList<>();
+        genotypeMap.add(SubSnpCoreFieldsTest.createGenotypeMap("GT", "0|1"));
+        genotypeMap.add(SubSnpCoreFieldsTest.createGenotypeMap("GT", "1|2"));
```

Comment:
I would add a coupe more genotypes to this test (1/1, 0/2...)

Reply:
And maybe some test with wrong genotypes to see we are catching them

Reply:
I've seen now that there are more complex genotypes tests in other class, so you can ignore those comments.